draft-ietf-taps-interface.md

```diff
@@ -2332,10 +2332,10 @@ a data transfer useful. It is given in bits per second. The special value -1 ind
 that no bound is specified.
 
 
-## TCP-specific Properties: User Timeout Option (UTO) {#tcp-uto}
+## TCP-specific Property: User Timeout Option (UTO) {#tcp-uto}
 
```

Comment:
```suggestion

Name:
: tcp.uto

Type:
: Numeric

```

Reply:
This is just the section heading. The actually configuration properties are defined below.

Reply:
sorry, my fault